usr/share/rear/lib/nbkdc-functions.sh

```diff
@@ -0,0 +1,54 @@
+# locate NovaBACKUP DC client configuration, referenced by
+#
+# prep/NBKDC/default/40_prep_nbkdc.sh
+# skel/NBKDC/etc/scripts/system-setup.d/59-start-nbkdc-client.sh
+
+
+# Set the NovaBACKUP DC default directory
+NBKDC_DIR=/opt/NovaStor/DataCenter
+
+
+function get_nbkdc_dir() {
+
+# check for running process rcmd-executor to determine its parent path
+NBKDC_DIR=$(ps -ef | grep rcmd-executor | grep -v grep | awk -F\  '{print $8}' | sed 's/\/rcmd-executor*//g')
```

Comment:
A running agent is a prerequisite and I need to find the location of the installed agent automatically. No, unfortunately we dont have a properties file in the /etc/ (yet), that is why that complex find statement. ;-)
Is pgrep a standard program in the most common distributions? I know in Ubuntu it is in the 'procps' package
pgrep wont display the base path of the executable, I would need to use awk and sed as well